Resources/Prototypes/_Floof/XenoArch/Effects/normal_effects.yml

```
@@ -105,3 +105,52 @@
       maxAmount: 1
   - type: ReplaceFloorArtifact
     replacement: FloorEighties
+
+- type: artifactEffect
+  id: EffectCashSpawnHigh
+  targetDepth: 2
+  effectHint: artifact-effect-hint-creation
+  components:
+  - type: SpawnArtifact
+    maxSpawns: 3
+    spawns:
+    - id: SpaceCash100
+      maxAmount: 5
+      prob: 0.75
+    - id: SpaceCash1000
+      maxAmount: 2
+      prob: 0.5
+    - id: SpaceCash5000
+      prob: 0.25
+    - id: SpaceCash10000
+      prob: 0.1
+
+- type: artifactEffect # Floof - A little radioactive gift
+  id: EffectSupermatterSliver
+  targetDepth: 3
```

This is a traitor objective, and in the future it can potentially become a dangerous item.
```suggestion
  targetDepth: 5
```

---

This is true, but also that same objective can spawn on maps without a Supermatter.
Also it generates less radiation then the demon core one.